src/pkcs11/framework-pkcs15.c

```diff
@@ -4064,10 +4167,11 @@ struct sc_pkcs11_object_ops pkcs15_prkey_ops = {
 	pkcs15_any_destroy,
 	NULL,	/* get_size */
 	pkcs15_prkey_sign,
-	NULL,	/* unwrap */
+	pkcs15_prkey_unwrap,
 	pkcs15_prkey_decrypt,
         pkcs15_prkey_derive,
-        pkcs15_prkey_can_do
+        pkcs15_prkey_can_do,
```

Comment:
please fix the indenting here and one line up.

Reply:
ok.